README.md

```diff
@@ -6,6 +6,28 @@
 
 Feature selection for scikit-learn estimators
 
+### Overview:
+If you have encountered a database with a myriad number of features, which could be messy to work on, a good idea is to approach this problem by selecting only some of these features for your model. Feature selection will reduce complexity, reduce the time when training an algorithm, and improve the accuracy of your model (if we select them wisely). However, this is not a trivial task. 
+
+To help you out performing this task, we have created the **feature-selection** package in `python`.
+
+If you are interested in a similar feature selection package for `R`, click [here](https://github.com/UBC-MDS/feature-selection-r).
+
+### Feature description:
+In this package, four functions are included to lead you with feature selection:
+
+#### Forward
+- Interative algorithm that starts as an empty model, add features that improves the accuracy of the model, and stops when the accuracy doesn't improve anymore.  
+
+#### Backward
+- TO DESCRIBE
+
+#### Variance Thresholding  
+- TO DESCRIBE
```

Comment:
Could you add the following description, please? Thanks! 

```suggestion
Select features based on their variances. A threshold, typically a low one, would be set so that any feature with a variance lower than that would be filtered out. Since this algorithm only looks at the features without the outputs, it could be used to do feature selection on data related to unsupervised learning.
```
Let me know if I could improve it further, although this algorithm is pretty straightforward.